test_scripts/Defects/7_1/1600_SetGlobalProperties_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to SetGlobalProperties component
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session are started
+-- 3) App is registered
+-- 4) App is activated
+--
+-- Steps:
+-- 1) Send SetGlobalProperties mobile RPC from app
```

Comment:
After step 1) and before step 2) we need to mention that SDL sends 2 requests to HMI